vcr/credential/util.go

```diff
@@ -136,6 +136,7 @@ func AutoCorrectSelfAttestedCredential(credential vc.VerifiableCredential, reque
 		}
 		if _, ok := credentialSubject[0]["id"]; !ok {
 			credentialSubject[0]["id"] = requester.String()
+			credential.CredentialSubject = make([]interface{}, 1)
```

Comment:
this is only a partial fix. If `didmetnods`config is `[web,nuts]` and the `credentialSubject.id` is already set by the user to the `did:nuts` address, it will still create a wallet with multiple DIDs as subject

Reply:
that was the idea: 3717